Fix authorization token Bearer prefix spelling in examples

srosenda opened the pull request:
the prefix should be spelled exactly as "Bearer", see RFC 6750, section 2.1. Authorization Request Header Field https://datatracker.ietf.org/doc/html/rfc6750#section-2.1.

Fixes #371.

bc-pi commented:
"BEARER" is perfectly legal https://www.rfc-editor.org/rfc/rfc9110.html#name-authentication-scheme

srosenda commented:
According to RFC9110 the authentication scheme is indeed case-insensitive contrary to how it is defined in RFC6750, so this issue can be closed. Would it still be good to unify the scheme spelling in the examples?

Sakurann commented:
@srosenda thanks a lot for doing the PR! Do you have contribution agreement/IPR signed with OIDF DCP WG?

srosenda commented:
> @srosenda thanks a lot for doing the PR! Do you have contribution agreement/IPR signed with OIDF DCP WG?

@Sakurann Thanks! I have not signed the contribution agreement. I am checking how to proceed on the matter.

Sakurann commented:
@srosenda you can always sign an individual contribution agreement. that would unblock us in a minute.

srosenda commented:
Rebased with main and amended the commit to unify the spelling also in the Token Response's `toke_type` parameter. Rewrote the commit message to state that this is not a correction, but an unification aimed to increase clarity.

srosenda commented:
> @srosenda you can always sign an individual contribution agreement. that would unblock us in a minute.

@Sakurann finally I should be covered by a contribution agreement signed by Digital and Population Data Services Agency Finland. Sorry for the delay.

srosenda commented:
Fixed terminology to refer to authorization scheme in the commit message and signed the commit.

bc-pi reviewed (approved):
Best I remember the use of "BEARER" was intentional to demonstrate to readers exactly the point that authentication scheme names are case-insensitive but maybe I'm not remembering correctly and maybe no one cares anymore or ever.

jogu commented:
Editorial, 4 approvals, open a while - merging! Thanks @srosenda !
